client/blocks/login/two-factor-authentication/security-key-form.jsx

```diff
@@ -31,8 +31,12 @@ class SecurityKeyForm extends Component {
 		isAuthenticating: false,
 	};
 
+	componentDidMount() {
+		setTimeout( () => this.initiateSecurityKeyAuthentication(), 100 );
+	}
+
 	initiateSecurityKeyAuthentication = ( event ) => {
-		event.preventDefault();
+		event?.preventDefault();
```

Comment:
This feels like a code smell, and a sign that we'll have to separate the event handling from what happens afterwards in 2 separate methods.

Reply:
Removed the `event.preventDefault()` call in 1fb7185df4 which as you point out does seem to be needed.

Reply:
Reverted the removal of `event.preventDefault` in [21517fd](https://github.com/Automattic/wp-calypso/pull/87128/commits/21517fdf68b3183f8f877b2df2cc51c7bd04e09c). This function is used on a form's `onSubmit` event, without it the form is submitted which causes a full page reload.